src/components/interactive-builder/add-question-modal.component.tsx

```diff
@@ -173,69 +173,6 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
     }
   };
 
-  const updateQuestion = (questionIndex) => {
-    try {
-      const mappedAnswers = selectedAnswers?.map((answer) => ({
-        concept: answer.id,
-        label: answer.text,
-      }));
```

Comment:
Has this been moved to somewhere else? or are u going for the duplicate question approach?

Reply:
This logic got moved to the `EditQuestionModal` component - so what's left over here is the cruft that ought to have been deleted.